.github/workflows/test-integration.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
 
       - name: Generate go test Slice
         id: test_split
-        uses: hashicorp-forge/go-test-split-action@v2
+        uses: hashicorp-forge/go-test-split-action@720a77701f0d1b2973126510492a6aad11ed0d5a # v2
```

Comment:
What is the benefit to use the hash instead of the tag?

The latest release is [here](https://github.com/hashicorp-forge/go-test-split-action/releases/tag/v2.0.0) and point to that commit

Reply:
It was not working with `v2` so I used commit linked to tag. Complete tag `v2.0.0` works, so I think we are good.